plume-models/src/blogs.rs

```diff
@@ -384,7 +387,7 @@ impl WithInbox for Blog {
     }
 
     fn is_local(&self) -> bool {
-        self.instance_id == 0
+        self.instance_id == 1 // TODO: this is not always true
```

Comment:
why was it 0 before and 1 now?

Reply:
This function is never used I think, so we didn't noticed this bug before, but the first ID in databases is 1, never 0.

Reply:
Instance should probably have a method that returns the local `instance_id`